meinberlin/apps/budgeting/assets/ListItemStats.jsx

```diff
@@ -19,19 +18,21 @@ export const ListItemStats = (props) => {
           >
             <i
               className="fa fa-chevron-up"
-              aria-label={positiveRatingsStr}
+              aria-hidden="true"
             />
-            {wrapSpaces(props.positiveCount)}
+            {props.positiveCount}
+            <span className="visually-hidden">{positiveRatingsStr}</span>
```

Comment:
aria-label is intended for interactive elements (https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label) so adding a sr span instead

Reply:
Oh! I always thought, the icons either need aria-hidden or aria-label. So, will remember this way to do it. And there might be more places in the code where that's wrong?

Reply:
@fuzzylogic2000 what do you mean? this has an aria hidden, also if you want to have an aria-label on an element, like in a button you can add role=img